### DISCUSSION TITLE
Share Volume Control

### DISCUSSION THREAD

**Discussion by Fluttrr** [navidrome/navidrome#2433] [Ideas]
Currently, shares have no volume control. This is obviously quite annoying for anyone who wants to listen to a share so I think it'd make sense to add one.

**Comment by ThatGuyThimo** [comment]
I would like to bump this, this shouldn't be a hard feature to implement and would make the share feature 100% more useful especially on desktop. not being able to adjust volume makes it unusable to some/ most people.

**Comment by RavingGrob** [comment]
I'm also here to voice my support for this feature, as it's essentially unusable for desktop usage.

**Comment by Shido-Itsuka** [comment]
Yep. I think it's just a simple update users need

**Comment by emptyynes** [comment]
<img width="736" height="920" alt="mrp-web" src="https://github.com/user-attachments/assets/beee9df0-372f-4031-af19-092e0b7b70b1" />

**Comment by ssshingetsu** [comment]
Really need this
